spec/classes/logstash_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
```

Comment:
Nice. It seems like this could be really useful for failing fast, before the much slower acceptance tests are even tried.

I like the idea of a _lightweight_ unit test suite, with the bulk of the testing happening at the system/acceptance level.